apps/studio/components/ui/EntityTypeIcon.tsx

```diff
@@ -67,7 +67,8 @@ export const EntityTypeIcon = ({
     <div
       className={cn(
         'flex items-center justify-center text-xs h-4 w-4 rounded-[2px] font-bold',
-        type === ENTITY_TYPE.FOREIGN_TABLE && 'text-yellow-900 bg-yellow-500',
+        type === ENTITY_TYPE.FOREIGN_TABLE &&
+          'text-warning-600/80 dark:text-yellow-900 bg-yellow-500',
```

Comment:
This fixes the text legibility on light mode. Using `text-yellow-1100` looks like an old banana. Gross.